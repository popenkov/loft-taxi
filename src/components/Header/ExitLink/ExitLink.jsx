import { NavLink } from '../../UI/NavLink';
import styles from './ExitLink.module.scss';

const ExitLink = ({ handleClick }) => {
  return (
    <div className={styles.logoutContainer}>
      <NavLink
        name="Выйти"
        color="secondary"
        handleClick={handleClick}
      ></NavLink>
    </div>
  );
};

export default ExitLink;
