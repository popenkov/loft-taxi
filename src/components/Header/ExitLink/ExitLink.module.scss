@import '../../../styles/variables.scss';
@import '../../../styles/mixins.scss';
@import '../../../styles/breakpoints.scss';

.logoutContainer {
  margin-left: 16px;

  @include mobile {
    margin-left: 0;
  }
}
