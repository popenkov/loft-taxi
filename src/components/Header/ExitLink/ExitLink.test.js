import { screen, fireEvent } from '@testing-library/react';
import { renderTestApp } from '../../../test/helpers/renderTestApp';
import ExitLink from './ExitLink';

describe('exit btn component', () => {
  let container;

  beforeEach(() => {
    container = renderTestApp(<ExitLink />, {
      route: '/',
      initialState: {},
    });
  });

  test('renders correctly', async () => {
    expect(<ExitLink />).toMatchSnapshot();
  });

  test('shows "Выйти"', () => {
    expect(container.getByText(/выйти/i)).not.toBeNull();
  });
});
