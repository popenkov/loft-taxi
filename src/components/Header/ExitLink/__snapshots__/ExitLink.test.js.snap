// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exit btn component renders correctly 1`] = `<ExitLink />`;
